tuna/fin_eval.py

```diff
@@ -219,8 +219,10 @@ def process_fdb_eval(self, fin_json, result_str='miopen_find_eval_result'):
         self.logger.info('Processing object: %s', fdb_obj)
         slv_stat = get_fin_slv_status(fdb_obj, 'evaluated')
         #retry returns false on failure, callback return on success
+        # pylint: disable=cell-var-from-loop
         ret = session_retry(session, self.update_fdb_eval_entry,
```

Comment:
is this complaining about fdb_obj? Can you look into rewriting this so we dont have to ignore the warning. @cderb  can help out

Reply:
done